css/_toolbars.scss

```diff
@@ -116,9 +116,9 @@
             }
         }
         i.disabled, .disabled i {
-            cursor: initial;
-            color: #fff;
-            background-color: #a4b8d1;
+            cursor: initial !important;;
+            color: #fff !important;;
+            background-color: #a4b8d1 !important;;
```

Comment:
Why do you need !important? What styling is taking higher priority?

Reply:
Yep it is .icon-mic-disabled 
![image](https://user-images.githubusercontent.com/3263098/59260040-cac9ff80-8c32-11e9-8b96-94daffe26b6e.png)

Maybe that one should be removed ...

Reply:
It seems to me that icon-mic-disabled is for muted ... and it always overrides the disabled css styling.